packages/relay-compiler/bin/RelayCompilerBin.js

```diff
@@ -94,6 +121,8 @@ async function run(options: {
   watch?: ?boolean,
   validate: boolean,
   quiet: boolean,
+  language: string,
```

Comment:
should we make this optional and default to `'javascript'`?

Reply:
It defaults to `javascript` in the option parsing phase, is there a reason to do it here instead?

Reply:
Ah got it, sorry missed that.